exporter/util.go

```diff
@@ -664,9 +708,15 @@ func (ic *importContext) getUpdatedSinceStr() string {
 }
 
 func (ic *importContext) getUpdatedSinceMs() int64 {
-	if ic.updatedSinceMs == 0 {
-		tm, _ := time.Parse(time.RFC3339, ic.updatedSinceStr)
-		ic.updatedSinceMs = tm.UnixMilli()
-	}
 	return ic.updatedSinceMs
 }
+
+func getEnvAsInt(envName string, defaultValue int) int {
+	if val, exists := os.LookupEnv(envName); exists {
+		parsedVal, err := strconv.Atoi(val)
+		if err == nil {
+			return parsedVal
+		}
```

Comment:
Potentially we should log an error if we couldn't parse the env variable. The user intended to set this but we couldn't parse it.